### ISSUE TITLE
AppImage freezing at first start on Ubuntu 20.04

### ISSUE THREAD

**Issue by blafasel42** [lensapp/lens#386]
**Describe the bug**
First i tried to install the SNAP with Desktop Installer. Installation went fine, but when i started the application, it just did not do anything.

Then downloaded the AppImage and started it from the Commandline.
Output was:
./Lens-3.4.0.AppImage 
CLUSTER STORE, MIGRATION: 2.0.0-beta.2
CLUSTER STORE, MIGRATION: 2.4.1
CLUSTER STORE, MIGRATION: 2.6.0-beta.2
CLUSTER STORE, MIGRATION: 2.6.0-beta.3
CLUSTER STORE, MIGRATION: 2.7.0-beta.0
CLUSTER STORE, MIGRATION: 2.7.0-beta.1

Then, nothing happened.

**To Reproduce**
1. Download AppImage 3.4.0 to Ubuntu 20.04
2. Start AppImage
3. See it freeze

**Expected behavior**
I expected some GUI to appear

**Screenshots**
no screen. thats my problem

**Environment (please complete the following information):**
- Lens Version: 3.4.0 (Snap and AppImage)
- OS: Ubuntu 20.04
- Installation method: AppImage and Snap
 
**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
./Lens-3.4.0.AppImage 
CLUSTER STORE, MIGRATION: 2.0.0-beta.2
CLUSTER STORE, MIGRATION: 2.4.1
CLUSTER STORE, MIGRATION: 2.6.0-beta.2
CLUSTER STORE, MIGRATION: 2.6.0-beta.3
CLUSTER STORE, MIGRATION: 2.7.0-beta.0
CLUSTER STORE, MIGRATION: 2.7.0-beta.1

When starting Snap i see in syslog:
May 22 12:07:04 grisslbeck systemd[2655]: tracker-extract.service: Succeeded.
May 22 12:07:05 grisslbeck kontena-lens[9702]: Locale not supported by C library.#012#011Using the fallback 'C' locale.
May 22 12:07:07 grisslbeck kontena-lens_kontena-lens.desktop[9702]: CLUSTER STORE, MIGRATION: 2.0.0-beta.2
May 22 12:07:07 grisslbeck kontena-lens_kontena-lens.desktop[9702]: CLUSTER STORE, MIGRATION: 2.4.1
May 22 12:07:07 grisslbeck kontena-lens_kontena-lens.desktop[9702]: CLUSTER STORE, MIGRATION: 2.6.0-beta.2
May 22 12:07:07 grisslbeck kontena-lens_kontena-lens.desktop[9702]: CLUSTER STORE, MIGRATION: 2.6.0-beta.3
May 22 12:07:07 grisslbeck kontena-lens_kontena-lens.desktop[9702]: CLUSTER STORE, MIGRATION: 2.7.0-beta.0
May 22 12:07:07 grisslbeck kontena-lens_kontena-lens.desktop[9702]: CLUSTER STORE, MIGRATION: 2.7.0-beta.1
May 22 12:07:24 grisslbeck tracker-store[9549]: OK
May 22 12:07:24 grisslbeck systemd[2655]: tracker-store.service: Succeeded.

```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
quite standard config with 1 cluster, 1 context, 1 user. no preferences
```

**Additional context**

**Comment by Nokel81** [comment]
Is this still a problem in 4.1.4?

**Comment by rsixel** [comment]
Still happening with version 4.2.2 and 4.2.4 on Ubuntu LTS 20.04 installing via snap.

**Comment by Nokel81** [comment]
Have you given write permissions to `$XDG_CONFIG_HOME/kontena-lens` or `~/.config/kontena-lens`?

**Comment by rsixel** [comment]
> Have you given write permissions to `$XDG_CONFIG_HOME/kontena-lens` or `~/.config/kontena-lens`?

Solved my issue installing version 5.0.0 BETA
